package.json

```diff
@@ -35,7 +35,7 @@
     "storybook": "start-storybook -p 6006",
     "test": "npx jest",
     "test:integration": "./scripts/test-integration",
-    "test:watch": "jest --watch",
+    "test-storybook": "test-storybook",
```

Comment:
Russ suggested we could run `npm run test-storybook -- --watch` and the same for test:watch locally instead.

Reply:
yeah to me those convenience methods are overkill but that may be just an opinion. Having it there resulted in Natalie thinking that adding a `test-storybook:watch` would also be necessary, because she's less familiar with jest and how it works.

It's convenient for sure but there is cognitive load and maintenance burden on everything we write. This may be inspiring a blog post.